.github/workflows/unittest.yml

```diff
@@ -0,0 +1,48 @@
+name: Unit tests
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
pylxd still uses 'master' not 'main'

Reply:
aah, that probably explains why it's not running right now :)